modules/apps/site/site-cms-site-initializer/src/main/java/com/liferay/site/cms/site/initializer/internal/display/context/ViewVersionHistoryDisplayContext.java

```diff
@@ -53,6 +56,24 @@ public List<FDSActionDropdownItem> getFDSActionDropdownItems() {
 				"{file.link.href}", "download", "download",
 				_language.get(_httpServletRequest, "download"), "get", null,
 				"link"),
+			new FDSActionDropdownItem(
+				StringBundler.concat(
+					_themeDisplay.getPortalURL(), _themeDisplay.getPathMain(),
+					GroupConstants.CMS_FRIENDLY_URL,
+					"/edit_content_item?objectEntryId={id}",
+					"&p_l_mode=read&p_p_state=", LiferayWindowState.POP_UP,
+					"&redirect=", _themeDisplay.getURLCurrent()),
+				"view", "view-content",
+				LanguageUtil.get(_httpServletRequest, "view"), null, null, null,
+				HashMapBuilder.<String, Object>put(
+					"objectEntryFolderExternalReferenceCode", "L_CONTENTS"
+				).build()),
+			new FDSActionDropdownItem(
+				StringPool.BLANK, "view", "view-file",
+				_language.get(_httpServletRequest, "view"), null, null, null,
+				HashMapBuilder.<String, Object>put(
+					"objectEntryFolderExternalReferenceCode", "L_FILES"
```

Comment:
This will only affect the files in the root folder, isn't it?

If a file is in a different folder it will have a different ERC

Reply:
Yes, I agree with you. I was thinking that the value was the object entry parent root folder . I am going to change it

Reply:
@robertoDiaz done.

I use the file attribute in order to distinguish between a file entry and content entry. But, I think, that it would be great to have a field to have the type of the entry, that is to say, if the entry is a content or a file.